include/OpenColorIO/OpenColorIO.h

```diff
@@ -3604,22 +3630,8 @@ class OCIOEXPORT BuiltinConfigRegistry
      */
     virtual bool isBuiltinConfigRecommended(size_t configIndex) const = 0;
 
-    /**
-     * @brief Get the default recommended built-in config.
-     * 
-     * Get the name of the built-in config that is currently recommended as the default config 
-     * to use for applications looking for basic color management. 
-     * 
-     * As the built-in config collection evolves, the default config name will change in future
-     * releases. 
-     * 
-     * For backwards compatibility, the name provided here will always work as an argument 
-     * to other methods so that any previous default config may be recovered.
-     * 
-     * Throws if the name is not found.
-     * 
-     * @return Default's built-in config name.
-     */
+    // Return the full forward-compatible name of the default built-in config.
+    OCIO_DEPRECATED("This was marked as deprecated starting in v2.2.2, please use ResolveConfigPath(\"ocio://default\").")
```

Comment:
You raise a good point Remi.  I agree that in general we would not want to do that.  In this case, the function that's being deprecated was just introduced a few months ago, so we felt that it would not cause anyone trouble (since I'm not aware of anyone that's shipped something that uses 2.2 yet).  But perhaps it would be better to just comment it as deprecated but hold off on using OCIO_DEPRECATED until 2.3.

Reply:
Agreed in this case it's more theoretical and shouldn't affect anyone good point.

Reply:
I'll remove it for now and leave a comment.